tests/AutoGourcerTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+declare(strict_types=1);
+
+include_once '../vendor/autoload.php';
+
+use \PHPUnit\Framework\TestCase;
+use \dje\AutoGourcer\AutoGourcer;
+
+/**
+ * Class AutoGourcerTestTest
+ *
+ * @source https://jtreminio.com/2013/03/unit-testing-tutorial-introduction-to-phpunit/ Thank you!!!
+ * @package tests
+ */
+class AutoGourcerTest extends TestCase
+{
+    /**
+     * @var null
+     */
+    protected $ag = null;
```

Comment:
I'd rather see this with a full name rather than abbreviation, and you can do `protected $ag;` here and basically have the same effect.